Tools/scripts/deepfreeze.py

```diff
@@ -249,14 +245,44 @@ def generate_code(self, name: str, code: types.CodeType) -> str:
         # Derived values
         nlocals, nplaincellvars, ncellvars, nfreevars = \
             get_localsplus_counts(code, localsplusnames, localspluskinds)
-        with self.block(f"static struct PyCodeObject {name} =", ";"):
-            self.object_head("PyCode_Type")
+        co_code_adaptive = make_string_literal(code.co_code)
+        self.write("static")
+        with self.indent():
+            with self.block("struct"):
+                self.write("PyObject_VAR_HEAD")
+                self.write("PyObject *co_consts;")
+                self.write("PyObject *co_names;")
+                self.write("PyObject *co_exceptiontable;")
+                self.write("int co_flags;")
+                self.write("int co_warmup;")
+                self.write("int co_argcount;")
+                self.write("int co_posonlyargcount;")
+                self.write("int co_kwonlyargcount;")
+                self.write("int co_stacksize;")
+                self.write("int co_firstlineno;")
+                self.write("int co_nlocalsplus;")
+                self.write("int co_nlocals;")
+                self.write("int co_nplaincellvars;")
+                self.write("int co_ncellvars;")
+                self.write("int co_nfreevars;")
+                self.write("PyObject *co_localsplusnames;")
+                self.write("PyObject *co_localspluskinds;")
+                self.write("PyObject *co_filename;")
+                self.write("PyObject *co_name;")
+                self.write("PyObject *co_qualname;")
+                self.write("PyObject *co_linetable;")
+                self.write("PyObject *co_endlinetable;")
+                self.write("PyObject *co_columntable;")
+                self.write("PyObject *co_weakreflist;")
+                self.write("void *co_extra;")
```

Comment:
I think this should be a definition in code.h so you don't have to update this every time the code object changes.